src/operator/cast-inl.h

```diff
@@ -0,0 +1,152 @@
+/*!
```

Comment:
Probably trailing space was removed by IDE.  Not sure why the diff looks like this. I checked the "real" file and it looks normal:/*!
 * Copyright (c) 2015 by Contributors
 * \file cast-inl.h
 * \brief cast operator
 * \author Junyuan Xie
*/
#ifndef MXNET_OPERATOR_CAST_INL_H_
.
.
.